### ISSUE TITLE
Event loop does not finish after client.close

### ISSUE THREAD

**Issue by timarandras** [deepstreamIO/deepstream.io-client-js#498]
```javascript
const deepstream = require('@deepstream/client');
const client = deepstream('localhost:6020');

async function start() {
  try {
    await client.login();

    const res = await client.rpc.make('echo', 'Bodza');
    console.log(res);
    
  } catch(e) {
    console.error(e);
  } finally {
    console.log('Closing');    
    client.close();
  }
}

start();
```

When client.close() is called the process keeps running.

See this codesandbox for demo:
https://codesandbox.io/embed/deepstream-v4-client-close-never-exits-yjnq0

Or this video:
https://streamable.com/gwgo3

Expected behaviour:
Process shall exit because the event loop has been exhausted.

deepstream.io-client-js@^3 don't has this issue.

@deepstream/client version: ^4.0.0-rc.41

node -v
v10.16.0

**Comment by timarandras** [comment]
https://streamable.com/gwgo3

**Comment by yasserf** [comment]
Great thanks! Probably the timer registry, will fix for next release!

**Comment by yasserf** [comment]
Fixed! In 4.0.0-rc.42
